src/client/Microsoft.Identity.Client/Instance/Discovery/KnownMetadataProvider.cs

```diff
@@ -28,9 +30,17 @@ void AddToKnownCache(InstanceDiscoveryMetadataEntry entry)
                 }
             }
 
+            void AddToPublicEnvironment(InstanceDiscoveryMetadataEntry entry)
```

Comment:
We tend to always specify the modifier, i.e. "private"

Reply:
This is inside the static constructor and access modifier cannot be applied here.